packages/@o3r/application/schemas/functional-content.metadata.schema.json

```diff
@@ -10,7 +10,6 @@
     "description",
     "sections"
   ],
-  "additionalProperties": false,
```

Comment:
Why removing strong typing on JSON check?

Reply:
I am going to sound crazy and seem afraid of you but do not get too excited :p.
After some discussions internally, we decided to cancel this PR.

FYI- the original reason:
This story has been raised after some discussions as the FC were not correctly used by some internal apps (That is why we now have the new property outputObject).
So the goal of this story was to let them work/test then add back the additionalProperties to false later.